src/run.ts

```diff
@@ -28,7 +28,8 @@ export async function run() {
       const resourceGroupName = core.getInput('resource-group', {
          required: true
       })
-      const clusterName = core.getInput('cluster-name', {required: true})
+      const resourceName = core.getInput('resource-name', {required: true})
+      const resourceType = core.getInput('resource-type') || ''
```

Comment:
```suggestion
      const resourceType = core.getInput('resource-type') || 'Microsoft.ContainerService/managedClusters'
```

Reply:
also add some validation that will fail if resourceType isn't recognised - i.e. is not a lower case match against microsoft.containerservice/{fleets,managedclusters} for now